docs/manual/src/internals/object_references.md

```diff
@@ -85,21 +99,4 @@ This indirection gives us some important safety properties:
   then the handlemap will throw an error with high probability, providing some amount of run-time typechecking
   for correctness of the generated bindings.
 * The handlemap can ensure we uphold Rust's requirements around unique mutable references and threadsafey,
-  using a combination of compile-time checks and runtime locking depending on the details of the underlying
-  Rust struct that implements the interface.
-
-## Managing Concurrency
-
-By default, uniffi uses the [ffi_support::ConcurrentHandleMap](https://docs.rs/ffi-support/0.4.0/ffi_support/handle_map/struct.ConcurrentHandleMap.html) struct as the handlemap for each declared instance. This class
-wraps each instance with a `Mutex`, which serializes access to the instance and upholds Rust's guarantees
-against shared mutable access.  This approach is simple and safe, but it means that all method calls
-on an instance are run in a strictly sequential fashion, limiting concurrency.
-
-For instances that are explicited tagged with the `[Threadsafe]` attribute, uniffi instead uses
-a custom `ArcHandleMap` struct. This replaces the run-time `Mutex` with compile-time assertions
-about the safety of the underlying Rust struct. Specifically:
-
-* The `ArcHandleMap` will never give out a mutable reference to an instance, forcing the
-  underlying struct to use interior mutability and manage its own locking.
-* The `ArcHandleMap` can only contain structs that are `Sync` and `Send`, ensuring that
-  shared references can safely be accessed from multiple threads.
+  by enforcing that the contained type is `Send+Sync`, and by refusing to hand out any mutable references.
```

Comment:
s/enforcing that the contained type is/specifying that the contained type must be/ or something? "enforcing" makes it sound like some kind of check the map itself is doing, but it's the compiler that is actually enforcing.